pkg/detector/library/driver.go

```diff
@@ -54,6 +54,9 @@ func NewDriver(libType string) (Driver, error) {
 		// Only semver can be used for version ranges
 		// https://docs.conan.io/en/latest/versioning/version_ranges.html
 		comparer = compare.GenericComparer{}
+	case ftypes.Cocoapods:
+		ecosystem = "cocoapods" // TODO use const from Trivy-db after adding it
+		comparer = compare.GenericComparer{}
```

Comment:
Trivy doesn't support CocodPods for vulnerability scanning. I wonder we should return unsupported error.

Reply:
Or, we can explicitly mention like "CocoaPods is supported for SBOM, not for vulnerability scanning".

Reply:
Added warning about this.